.github/workflows/backport.yaml

```diff
@@ -0,0 +1,50 @@
+name: Backport
+on:
+  pull_request_target:
+    types: [closed, labeled]
+
+# WARNING:
+# When extending this action, be aware that $GITHUB_TOKEN allows write access to
+# the GitHub repository. This means that it should not evaluate user input in a
+# way that allows code injection.
+
+jobs:
+  backport:
+    name: Backport Pull Request
+    # Run the action if a PR is merged with backport labels
+    # OR
+    # when already merged PR is labeled with backport labels
+    if: >
+      github.event.pull_request.merged
+      && (
+        github.event.action == 'closed'
+        || (
+          github.event.action == 'labeled'
+          && startsWith(github.event.label.name, 'backport')
```

Comment:
We could probably add a slash here: `'backport/'`. This would match the regex on line 46, and it would also not trigger on the "backport" label: https://github.com/k0sproject/k0s/labels?q=backport

Reply:
done